### PR TITLE
data dump and footer model

### PR DESCRIPTION
**Pull request by RachellCalhoun**
Let me know if I did this right. I made a json file wtih data dump. I was in the process of doing a footer model but Couldn't migrate. So I wanted to get the data out first then fix my db.

### CODE REVIEW

**Comment by RachellCalhoun** [comment]
@jinpark :)

**Comment by jinpark** [comment]
can you split the PR up, one for the datadump and one for the footer model?

**Comment by RachellCalhoun** [comment]
I deleted the json file.

**Comment by RachellCalhoun** [comment]
@jinpark  can you have a look?
footer2
footer1

File: content/static/css/content.css [hunk 1]
```diff
@@ -65,6 +67,7 @@ body {
 .white-text {
   color: #fff;
 }
+.
```
Comment (jinpark) [inline content/static/css/content.css]: ?

File: content/views.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 from django.shortcuts import render
-from .models import OurStory, OurTeam, EduProgram, EthicalPost, VolunteerPeru, VolunteerOpenPosition, VolunteerAbout, CustomPage
+from .models import OurStory, OurTeam, EduProgram, EthicalPost, VolunteerPeru, VolunteerOpenPosition, VolunteerAbout, CustomPage, FooterInfo
```
Comment (jinpark) [inline content/views.py]: i guess you dont need footerinfo here anymore?

**Pull request merged**